pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/AdminProxyHandler.java

```diff
@@ -122,7 +125,16 @@ protected HttpClient newHttpClient() {
     protected String rewriteTarget(HttpServletRequest request) {
         StringBuilder url = new StringBuilder();
 
-        if (isBlank(brokerWebServiceUrl)) {
+        boolean isFunctionsRestRequest = false;
```

Comment:
maybe move this to a method isFunctionsRequest(HttpServletRequest request)?
or 
boolean isFunctionsRestRequest = requestUri.startsWith("/admin/v2/functions") || requestUri.startsWith("/admin/functions");